app/controllers/api/subcollections/security_groups.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module Subcollections
+    module SecurityGroups
+      def security_groups_query_resource(object)
+        object.respond_to?(:security_groups) ? object.security_groups : []
```

Comment:
Can you add a test that executes the case when the object does not respond to `:security_groups`?

Reply:
Done - thanks!